source/network/packets/StartGamePacket.cpp

```diff
@@ -16,33 +16,39 @@ void StartGamePacket::handle(const RakNet::RakNetGUID& guid, NetEventCallback* p
 void StartGamePacket::write(RakNet::BitStream* bs)
 {
 	bs->Write((unsigned char)PACKET_START_GAME);
-	bs->Write(field_4);
-	bs->Write(field_8);
-	bs->Write(field_C);
-	bs->Write(field_10);
-	bs->Write(field_14);
-	bs->Write(field_18);
+	bs->Write(m_seed);
+	bs->Write(m_levelVersion);
+	bs->Write(m_entityId);
+#ifdef TEST_GAMEMODE_REPLICATION
```

Comment:
I ordered it like this because Mojang themselves had GameType in a similar location in the struct, and because this is networking, the order in which this information is sent matters.

Reply:
See conversation in Packet.hpp.